### ISSUE TITLE
Word guess for 41:49 missing word.

### ISSUE THREAD

**Issue by StevenMaude** [poppingtonic/greenwald-30c3-keynote#18]
It might be "tyrannies", though that's pretty emotive (maybe that was the intent...). Not completely sure, so didn't include in my pull request.

**Comment by poppingtonic** [comment]
You're right, the word is "tyrannies". I added it thanks to a pull request (issue #19), by @bgyss.
